run_coder.py

```diff
@@ -0,0 +1,45 @@
+import torch
+import typer
+from pathlib import Path
+
+import settings
+import model
+from inference import INF
+
+
+def main(
+    path: Path = typer.Argument(
+        ...,
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        readable=True,
+        resolve_path=True,
+        help="Path to file with fens",
+    ),
+    modelPath: Path = typer.Option(
+        ...,
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        readable=True,
+        resolve_path=True,
+        help="Path to file with configuration enviromental variables",
+    ),
+):
+    model = model.Coder(settings.BOARD_SHAPE, settings.LATENT_SIZE).to(settings.DEVICE)
+    print(model_path)

```

Comment:
if you want to leave this print in the final code, besides model path print also what you are printing, so that the user will see what is happening